### PR TITLE
Dashing Patch Release 8

### LINKED ISSUE #986: Dashing Patch Release 8
This issue will track the broad status of the next Dashing patch release. The date for this patch release is not currently set and will be determined by a critical mass of desired changes ready to be released.

We're going to use a GitHub Project board for this patch release again: [Dashing Patch Release 8](https://github.com/orgs/ros2/projects/24)

As with prior releases, if you want to contribute fixes please open pull requests targeting the latest development branch and mention to the reviewer that you'd like the pull request to be considered for backporting into Dashing.

### PR DESCRIPTION
**Pull request by nuclearsandwich**
Version updates for dashing patch release 8.

Closes #986.

### CODE REVIEW

**Comment by nuclearsandwich** [comment]
* Ubuntu [Build Status](http://ci.ros2.org/job/ci_linux/13186/)
* CentOS [Build Status](https://ci.ros2.org/job/ci_linux-centos/89/)
* Ubuntu-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/8118/)
* Ubunty-armhf [Build Status](https://ci.ros2.org/job/ci_linux-armhf/82/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/13200/)
* Windows Debug [Build Status](http://ci.ros2.org/job/ci_windows/13201/)

**Comment by nuclearsandwich** [comment]
The CI above includes no new regressions over previous versions of Dashing.

**Review by gbiggs — APPROVED** [review]

File: ros2.repos [hunk 1]
```diff
@@ -226,15 +226,15 @@ repositories:
   ros2/rmw_connext:
     type: git
     url: https://github.com/ros2/rmw_connext.git
-    version: 0.7.5
+    version: 0.7.6
   ros2/rmw_cyclonedds:
     type: git
     url: https://github.com/ros2/rmw_cyclonedds.git
-    version: 0.5.1
+    version: 0.7.0
```
Comment (clalancette) [inline ros2.repos:233]: Just double-checking here; the latest tag in the `0.7` "series" looks to be `0.7.5`.  Is this correct?

**Review by clalancette — COMMENTED** [review]
I actually have one inline question.

File: ros2.repos (hunk 1 above)
Comment (nuclearsandwich) [reply ros2.repos:233]: This is the version of CycloneDDS that was released into Dashing by their team.

**Review by mjcarroll — APPROVED** [review]

**Review by jacobperron — APPROVED** [review]
:tada:

**Review by clalancette — APPROVED** [review]

**Review by sloretz — APPROVED** [review]

**Pull request merged**